modules/ROOT/nav.adoc

```diff
@@ -30,6 +30,7 @@
 ** xref:cloud-guides:use-teradata-vantage-with-azure-machine-learning-studio.adoc[]
 ** xref:cloud-guides:integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc[]
 ** xref:other-integrations:integrate-teradata-vantage-with-knime.adoc[]
+** xref:studio.mac.m1.help.adoc[]
```

Comment:
To improve SEO, it's best if you use a more descriptive file name with hyphens, e.g. `install-teradata-studio-on-mac-m1-m2.adoc`